examples/iframe/observer.mjs

```diff
@@ -0,0 +1,6 @@
+let data;
+function refresh() {
+    data = new observer.Observer({});
```

Comment:
Can't you use the module version of `Observer`?

Reply:
Yes that is true I will add this too